modules/core/client/css/lightbox.css

```diff
@@ -26,3 +26,17 @@
     left:auto;
     right: 0;
 }
+
+.lightbox-app {
+  position: fixed;
+  /*max-height: 500;*/
+  top: 5%;
+  left: 10%;
+  right: 10%;
+  padding: 10px;
+  background-color: white;
+  border: 2px solid black;
+  z-index: 2;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
```

Comment:
I removed the lightbox-app. When I did this, we were still using the other application and needed to mess around with how it fit inside the form. However, since we're back to using a standard form, this isn't needed.